docs/tutorials/rerank/rerank_pipeline_with_Amazon_Rerank_model_on_Amazon_Bedrock.md

```diff
@@ -0,0 +1,622 @@
+# Topic
+
+[Reranking pipeline](https://opensearch.org/docs/latest/search-plugins/search-relevance/reranking-search-results/) is a feature released in OpenSearch 2.12.
+It can rerank search results, providing a relevance score for each document in the search results with respect to the search query.
+The relevance score is calculated by a cross-encoder model. 
+
+This tutorial explains how to use the [Amazon Rerank 1.0 model in Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/rerank-supported.html) in a reranking pipeline. 
+
+Note: Replace the placeholders that start with `your_` with your own values.
+
+# Steps
+
+## 0. Test the model on Amazon Bedrock
+You can perform a reranking test with the following code.
+
+```python
+import json
+import boto3
+bedrock_region = "your_bedrock_model_region_like_us-west-2"
+bedrock_runtime_client = boto3.client("bedrock-runtime", region_name=bedrock_region)
+
+modelId = "amazon.rerank-v1:0"
+contentType = "application/json"
+accept = "*/*"
+
+body = json.dumps({
+    "query": "What is the capital city of America?",
+    "documents": [
+        "Carson City is the capital city of the American state of Nevada.",
+        "The Commonwealth of the Northern Mariana Islands is a group of islands in the Pacific Ocean. Its capital is Saipan.",
+        "Washington, D.C. (also known as simply Washington or D.C., and officially as the District of Columbia) is the capital of the United States. It is a federal district.",
+        "Capital punishment (the death penalty) has existed in the United States since beforethe United States was a country. As of 2017, capital punishment is legal in 30 of the 50 states."
+    ]
+})
+
+response = bedrock_runtime_client.invoke_model(
+    modelId=modelId,
+    contentType=contentType,
+    accept=accept, 
+    body=body
+)
+results = json.loads(response.get('body').read())["results"]
+print(json.dumps(sorted(results, key=lambda x: x['index']),indent=2))
```

Comment:
How about just print the model response directly? Adding `sorted` may confuse user as this is some processed resposne.

Reply:
```
print(json.dumps(results, indent=2))
```
Response
```
[
  {
    "index": 2,
    "relevance_score": 0.7711548724998493
  },
  {
    "index": 0,
    "relevance_score": 0.0025114635138098534
  },
  {
    "index": 1,
    "relevance_score": 2.4876490010363496e-05
  },
  {
    "index": 3,
    "relevance_score": 6.339210403977635e-06
  }
]
```

Reply:
Thanks, I'm going to updated based on your recommendation on next commit.

Reply:
fixed on 9482b472b2bc68bba61267c87f484967362e2c5b